dts/xtensa/intel/intel_adsp_ace30_ptl.dtsi

```diff
@@ -439,22 +439,10 @@
 				compatible = "intel,adsp-power-domain";
 				bit-position = <15>;
 			};
-			ml1_domain: ml1_domain {
-				compatible = "intel,adsp-power-domain";
-				bit-position = <13>;
```

Comment:
Nitpick: you just went through the trouble to adjust the bit-position field three patches back, and now you're removing it?  Doing it in the opposite order would be clearer, with less churn.

Reply:
This is a valid point :) For some reason, I planned to organize these power domains first and clean them up later. In my defense, it's already Friday 5:46 PM in my time zone.

Best regards.